language/message/fr.po

```diff
@@ -1211,12 +1211,19 @@ msgstr "Aucun résultat trouvé."
 msgid "Please zoom in the map to see the results for this category."
 msgstr "Veuillez zoomer sur la zone pour voir des résultats."
 
-msgid "Please zoom out of the map."
-msgstr "Veuillez agrandir la zone de recherche."
-
 msgid "Search around my position"
 msgstr "Rechercher autour de moi"
 
+msgid "Hmm, looks like a no-man's land 🏜️"
+msgstr "Hum, il n’y a pas un chat ici 🐱"
+
+msgid "We found no place matching your query in this area."
+msgstr ""
+"Nous ne trouvons aucun lieu correspond à votre recherche dans cette zone."
```

Comment:
Oups, there is a missing word or word part in this string…

Reply:
@bbecquet i thought too but checked and fr.po has many occurrences of translated strings starting with "" \n 
It's okay :)

Reply:
oh, sorry I thought you were referring to the leading "" string on line 1221.
I vote for "correspodant à".